### PR TITLE
Add skeleton loader

### LINKED ISSUE #98: Implement skeleton loader in the frontend
Implement skeleton loader in the frontend: https://github.com/willmendesneto/ngx-skeleton-loader

### PR DESCRIPTION
**Pull request by MoritzWeber0**
# Description

This PR changes the loading animation. The goal is to have a unified and consistent loading animation in the platform. There is general component that can be included in all components that use the mat-card-overview. More loading animations can be added in the future. 
image
Resolves #98

### CODE REVIEW

File: frontend/src/app/projects/project-overview/project-overview.component.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export class ProjectOverviewComponent implements OnInit {
       .list()
       .pipe(
         tap(() => {
-          this.showSpinner = false;
+          this.loading = false;
         })
```
Comment (dominik003) [inline frontend/src/app/projects/project-overview/project-overview.component.ts:37]: Here the curly brackets can be removed and the `next` and `error` case of the subscription are the same (which should not be the case IMO

File: frontend/src/app/helpers/skeleton-loaders/mat-card-overview-loader/mat-card-overview-loader.component.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<ng-container *ngIf="loading">
+  <ngx-skeleton-loader
+    *ngFor="let _ of [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]"
```
[the remark this answers is not in the document]
Comment (MoritzWeber0) [reply frontend/src/app/helpers/skeleton-loaders/mat-card-overview-loader/mat-card-overview-loader.component.html]: Done. Doesn't work perfectly, but I was not able to find the exact route cause. But it's not important enough to spend more time there.

File: frontend/src/app/projects/project-overview/project-overview.component.ts (hunk 1 above)
Comment (MoritzWeber0) [reply frontend/src/app/projects/project-overview/project-overview.component.ts:37]: Removed the curly brackets. `next` and `error` are not the same on my side.

Comment (dominik003) [reply frontend/src/app/projects/project-overview/project-overview.component.ts]: Yes that was my mistake somehow I have seen no difference

**Review by dominik003 — APPROVED** [review]

**Pull request merged**